playbooks/common/openshift-cluster/upgrades/pre/verify_docker_upgrade_targets.yml

```diff
@@ -1,23 +1,20 @@
 ---
-- name: Verify docker upgrade targets
-  hosts: oo_masters_to_config:oo_nodes_to_upgrade:oo_etcd_to_config
-  tasks:
-  # Only check if docker upgrade is required if docker_upgrade is not
-  # already set to False.
-  - include: ../docker/upgrade_check.yml
-    when: docker_upgrade is not defined or docker_upgrade | bool and not openshift.common.is_atomic | bool
+# Only check if docker upgrade is required if docker_upgrade is not
+# already set to False.
+- include: ../docker/upgrade_check.yml
+  when: docker_upgrade is not defined or docker_upgrade | bool and not openshift.common.is_atomic | bool
 
-  # Additional checks for Atomic hosts:
+# Additional checks for Atomic hosts:
 
-  - name: Determine available Docker
-    shell: "rpm -q --queryformat '---\ncurr_version: %{VERSION}\navail_version: \n' docker"
-    register: g_atomic_docker_version_result
-    when: openshift.common.is_atomic | bool
+- name: Determine available Docker
```

Comment:
I have not modified the file, just remove the play part and kept all tasks. I can polish the task files if needed. However, we should fix this as a part of another PR, e.g. "towars atomic compatibility".